meta-sokol-flex-distro/conf/distro/sokol-flex.conf

```diff
@@ -94,16 +91,6 @@ APPEND_KGDBOC = "kgdbwait kgdboc=${CMD_SERIAL_CONSOLE}"
 
 # Splash screen
 SPLASH:mel ?= "psplash"
```

Comment:
still using mel here?

Reply:
there's also SDK_DEPLOY:mel in the same file.

Reply:
SDK_DEPLOY at least must use an override as it's defined later on and will overwrite any direct definition. I suspect that was the case for splash at some point too, but will check.